tests/overlay_tests.cpp

```diff
@@ -271,7 +271,7 @@ class OverlayTest : public MesosTest
            "rm",
            OVERLAY_NAME, 
            OVERLAY_NAME_2});
-      AWAIT_READY(docker);
+      docker.await();
```

Comment:
Why this change?

Reply:
many tests cases use only one of the overlay network and not both. In those cases, AWAIT_READY fails when it tries to remove the non-existing overlay network

Reply:
ic. Can you add some comments?